app/assets/javascripts/discourse/app/widgets/quick-access-profile.js

```diff
@@ -84,7 +84,9 @@ createWidgetFrom(QuickAccessPanel, "quick-access-profile", {
       {
         icon: "pencil-alt",
         href: `${this.attrs.path}/activity/drafts`,
-        content: I18n.t("user_action_groups.15"),
+        content: `${I18n.t("user_action_groups.15")} (${
```

Comment:
Maybe this widget already takes this into account, but does this break if `this.currentUser` is null (like in the case of anon?)

Reply:
It would also be good to not output `(0)` when the user has no drafts.

Reply:
Ok I added a check to not have `(0)` when no drafts are present. 

@martin-brennan I do not think this menu is even visible if this.currentUser is null.

I do however see an issue that the site needs a refresh before displaying the correct amount of drafts after drafts are deleted/created.

Is there a way to update this in the widget upon draft creation or deletion?